packages/carbon-graphs/src/js/helpers/label.js

```diff
@@ -336,6 +336,29 @@ const destroyTooltipDiv = () => {
   }
 };
 
+/**
+ * Update shapes in y and y2 axes during reflow.
+ *
+ * @private
+ * @param {object} graph - Graph instance
+ * @param {object} graphData - Graph data
+ * @param {Line} control - Line instance
```

Comment:
nit:  is this really a Line?  Looks to be central to multiple graphs.  Should this be updated?

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/94/commits/98f1f881fd37737792f72504fb242b88800e2f7d#diff-a75e9754db9269401928d444c9f036bc068af538ee5ca551e761767a5394e23eR345).